rich_argparse.py

```diff
@@ -4,7 +4,7 @@
 import re
 from typing import TYPE_CHECKING, Callable, Generator, Iterable, Tuple
 
-# rich is only used to display help. It is imported inside the functions in order
+# rich is only used to display help. It is imported inside the functions in order to
```

Comment:
This was correct, please don't change it.

Reply:
👍

Reply:
my grammar was wrong but "in order to not" is more idiomatic english, so i made that change.